cpanfile

```diff
@@ -11,6 +11,7 @@ requires 'ExtUtils::MakeMaker', '6.72'; # [1]
 requires 'Guard', '1.023';
 requires 'PadWalker', '2.2';
 requires 'aliased', '0.34';
+requires 'Net::SSLeay', '1.81';
```

Comment:
I've added a commit to upgrade that and SOAP::Lite, which then drops Crypt::SSLeay entirely. Not sure the SOAP::Lite using things are well tested (Zurich geocoder, Angus/EastHants report submission) so might need careful deploy though nothing major in changelog I can see.